### PR TITLE
Update student finance calculator with 2022/2023 data

### PR DESCRIPTION
**Pull request by hannako**
- Trello: https://trello.com/c/CaPrBLgh/989-fc-asap-live-18-01-student-finance-calculator-updates

- New data (supplied by content colleagues) 
https://docs.google.com/spreadsheets/d/1bmcpDQwgtxTNOLGC0hRQqOSOJRHrIxBD9pHxG-NR3kU/edit#gid=2113480503

- Review app: https://smart-answer-update-stu-29fbje.herokuapp.com/student-finance-calculator

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: lib/smart_answer/calculators/student_finance_calculator.rb [hunk 1]
```diff
@@ -14,88 +14,88 @@ class StudentFinanceCalculator
...
         "2021-2022" => {
           "at-home" => 1902,
           "away-in-london" => 3558,
           "away-outside-london" => 2534,
         },
+        "2022-2023" => {
+          "at-home" => 1902,
+          "away-in-london" => 3558,
+          "away-outside-london" => 2534,
+        },
```
[the remark this answers is not in the document]
Comment (hannako) [reply lib/smart_answer/calculators/student_finance_calculator.rb:39]: I believe that's intentional - in tab [2022 changes](https://docs.google.com/spreadsheets/d/1bmcpDQwgtxTNOLGC0hRQqOSOJRHrIxBD9pHxG-NR3kU/edit#gid=1738138405) on the spreadsheet, it has **n/a** for 2022-2023 `REDUCED_MAINTENTANCE_LOAN_AMOUNT`. I will double check with the content person though thanks.

File: test/unit/calculators/student_finance_calculator_test.rb [hunk 2]
```diff
@@ -209,84 +209,84 @@ class StudentFinanceCalculatorTest < ActiveSupport::TestCase
...
+            should "be £307.95" do
               calculator = StudentFinanceCalculator.new(
                 course_start: current_year,
                 household_income: 25_000,
                 residence: :unused_variable,
               )
-              assert_equal 298.69, calculator.childcare_grant_more_than_one_child
+              assert_equal 307.95, calculator.childcare_grant_more_than_one_child
             end
           end
```
[the remark this answers is not in the document]
Comment (hannako) [reply test/unit/calculators/student_finance_calculator_test.rb:236]: For 2021-2022 `CHILD_CARE_GRANTS_MORE_THAN_ONE_CHILD_HOUSEHOLD_INCOME = 27_499.55` , which is higher than the household income you entered (£25,000). So isn't that expected? Or am I missing something...

File: test/unit/calculators/student_finance_calculator_test.rb [hunk 3]
```diff
@@ -209,84 +209,84 @@ class StudentFinanceCalculatorTest < ActiveSupport::TestCase
         end
       end
 
-      context "in 2020-2021" do
-        current_year = "2020-2021"
+      context "in 2021-2022" do
+        current_year = "2021-2022"
```
[the remark this answers is not in the document]
Comment (hannako) [reply test/unit/calculators/student_finance_calculator_test.rb:213]: You are quite right! Thank you. I will add the new tests for 2022-2023 as a fresh commit so it's easier to review, and fixup after.

File: test/unit/calculators/student_finance_calculator_test.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (jon-kirwan) [reply test/unit/calculators/student_finance_calculator_test.rb:236]: Sorry my mistake. I was a bit confused with the way the ranges work for childcare grants but can see now that it's all working fine. Thanks

**Review by jon-kirwan — APPROVED** [review]
Thanks Jess. I've had a good look through, checked all the tests and various different scenarios from the form itself, and everything looks good to me. 👍

**Pull request merged**